com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -1559,15 +1559,32 @@ ref _cullingResults
                             visibleProbe.ForceRenderingNextUpdate();
                         }
 
-                        hdCamera.parentCamera = parentCamera; // Used to inherit the properties of the view
+                        hdCamera.SetParentCamera(hdParentCamera); // Used to inherit the properties of the view
+                        hdCamera.resetPostProcessingHistory = false;
```

Comment:
Why do we need this reset to off?

Reply:
hmmm not sure. I think it might be a typo. Testing without it

Reply:
Actually this is needed, but is done the wrong way. resetPostProcessingHistory is always true on probe cameras, so without this it would return always an invalid exposure texture.
The correct fix is to ignore this flag if the exposure texture is inherited from the parent. Will fix this right away.